extensions/podcasts-now/src/podcast-list.tsx

```diff
@@ -0,0 +1,133 @@
+
+    const newFeeds = JSON.parse(feedsValue).filter((url: string) => url !== feed);
+    await LocalStorage.setItem(PODCASTS_FEEDS_KEY, JSON.stringify(newFeeds));
+    setRemovedFeeds((prev) => [...prev, feed]);
+  };
+
+  const progress = statusData ? formatProgress(statusData.position, statusData.time) : "";
+  const statusTitle = statusMapping.titles[statusData?.state || PlayerState.stopped];
+  const controlActionWithRevalidateStatus = (action: () => Promise<unknown> | unknown) => {
+    return async () => {
+      setIsLoading(true);
+      await action();
+      revalidateStatus();
+      setIsLoading(false);
+    };
+  };
+
+  const trackName = statusData?.name || "";
+
+  const isPlayerLoading = isLoading || isStatusLoading;
+
+  if (filteredList.length === 0) {
+    return (
+      <List>
+        <List.EmptyView
+          icon={Icon.Cog}
+          title="No podcasts"
+          description="Please manage your subscribed podcasts first."
+          actions={
+            <ActionPanel>
+              <Action icon={Icon.Cog} title="Manage Podcasts" onAction={() => push(<AddPodcast />)} />
+            </ActionPanel>
+          }
+        />
+      </List>
+    );
+  }
+
+  return (
+    <List
+      isLoading={isLoading || isPodcastsLoading || isStatusLoading}
+      navigationTitle="Search Podcasts"
+      searchBarPlaceholder="Search your subscribed podcasts"
+    >
+      <List.Section title="Player">
+        <List.Item
+          icon={statusMapping.icons[statusData?.state || PlayerState.stopped]}
+          title={isPlayerLoading ? "Waiting..." : statusTitle}
+          subtitle={trackName}
+          accessories={[{ text: progress }]}
+          actions={
+            isPlayerLoading ? null : (
+              <ActionPanel title="Player Control">
+                <Action
+                  icon={statusData?.state === PlayerState.paused ? Icon.Play : Icon.Pause}
+                  title={statusData?.state === PlayerState.paused ? "Resume" : "Pause"}
+                  onAction={controlActionWithRevalidateStatus(togglePause)}
+                />
+                <Action
+                  icon={Icon.RotateClockwise}
+                  title="Forward 15s"
+                  onAction={controlActionWithRevalidateStatus(forward)}
+                />
+                <Action
+                  icon={Icon.RotateAntiClockwise}
+                  title="Rewind 15s"
+                  onAction={controlActionWithRevalidateStatus(rewind)}
+                />
+                <Action icon={Icon.Cog} title="Manage Podcasts" onAction={() => push(<AddPodcast />)} />
+              </ActionPanel>
+            )
+          }
+        />
+      </List.Section>
+      <List.Section title="Podcasts">
+        {filteredList.map((item) => {
+          return (
+            <List.Item
+              id={item.feedUrl}
+              key={item.feedUrl}
+              icon={{
+                source: item?.itunes?.image || "",
```

Comment:
Could you return a default icon to show? It looks empty on my end.

<img width="1000" alt="podcasts-now 2023-02-08 at 11 11 59" src="https://user-images.githubusercontent.com/16003285/217500298-fec1f5c0-649e-4d0f-af51-c3664e7e52c6.png">

Reply:
Add a default icon through the `fallback` attr